src/components/BookItem.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const BookItem = () => (
```

Comment:
- [ ] This component should also add a delete book button to each book, please add it to comply with the project requirements. (It doesn't have to work right now, just has to be present)